llvm/test/Instrumentation/AddressSanitizer/remove-memory-effects.ll

```diff
@@ -0,0 +1,23 @@
+; Remove possible memory effects from functions that are invalidated by
+; AddressSanitizer instrumentation.
+
+; RUN: opt -passes='asan<use-after-scope>' -S %s | FileCheck %s
+
+target datalayout = "e-i64:64-f80:128-s:64-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
Is this needed for the test to pass? If so, it will fail if the X86 backend hasn't been built. If the backend is needed, it needs REQUIRES:...